chart/Chart.yaml

```diff
@@ -15,14 +15,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.0.0-dev
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: latest
-
-dependencies:
-  - name: common
-    repository: "file://../../installer/manifests/common"
-    version: 0.1.0
```

Comment:
If we can publish the common chart in a helm repo or dedicated git repo, I'm happy to include it as it reduces code duplication.
I didn't want to copy the installer common chart into helm-service, jmeter-service (and consequently also other repos in keptn-contrib) as it includes way more than actually needed.

Reply:
For now, the common chart is released with keptn but it's not really advertised yet. The plan is to move it into a separate github repo at some point. But you can already include it directly from our helm index if you want to, but I understand if you wanna include it later when it's more mature or even never since it has lots of unneeded stuff for you

Reply:
I will try if I can include it from the helm chart repo directly.